nautobot/core/testing/views.py

```diff
@@ -780,6 +785,10 @@ def test_list_objects_anonymous(self):
         @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
         def test_list_objects_filtered(self):
             instance1, instance2 = self._get_queryset().all()[:2]
+            if hasattr(self.model, "name") and instance1.name == instance2.name:
```

Comment:
Fixing issues with random data generation sometimes creating two instances with the same name

Reply:
I think the issue might actually be where we're creating data that looks realistic like `Interface 1` - `Interface 24` on more than one device.